### ISSUE TITLE
Working with business days 

### ISSUE THREAD

**Issue by amurguiag** [facebook/prophet#1445]
I have a daily frequency data set that only contains business days. Do I still need to include holidays into the model? Also, should all the _period_ parameters in add_seasonality() be in terms of business days? Example: if I add weekly seasonality, this should be period = 251/52 as there are about 251 business days in a year.  

Have a great day!

**Comment by bletham** [comment]
No, the underlying model is continuous time, so the data frequency doesn't matter and you don't need to make any changes to the model parameters based on data frequency. So weekly seasonality will still be 7 days. The only thing that would be important here is that the model won't be able to make good predictions for weekend days (because there's no weekend data in the history), so you'll want to make sure that you leave out weekend days from the dataframe you pass into `predict`. There's some documentation on that matter here: https://facebook.github.io/prophet/docs/non-daily_data.html#data-with-regular-gaps

As for holidays, you can leave them in and basically only holidays that fall on weekdays will be included in the model.
